collections.yml

```diff
@@ -78,3 +79,25 @@ James Bond:
   - Quantum of Solace
   - Skyfall
   - Spectre
+American Pie:
+  - American Pie
+  - American Pie 2
+  - American Reunion
+  - American Wedding
+Cars:
+  - Cars
+  - Cars 2
+  - Cars 3
+John Wick:
+  - John Wick
+  - 'John Wick: Chapter 2'
+Now You See Me:
+  - Now You See Me 1
+  - Now You See Me 2
+Kingsman:
```

Comment:
For the Kingsman movies, replace `:` with `.+?` so that this catches a colon, dash, or simply a space that may be between the title and the subtitle